subsys/bluetooth/audio/Kconfig.otc

```diff
@@ -0,0 +1,34 @@
+# Bluetooth Audio - Object transfer configuration options
+
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if BT_AUDIO
+
+#### Object Transfer Service Client ################################
+
+config BT_OTC
+	bool "Object Transfer Service Client Support"
+	select BT_L2CAP_DYNAMIC_CHANNEL
+	select BT_GATT_CLIENT
+	select BT_GATT_AUTO_DISCOVER_CCC
+	default n
+	help
+	  This option enables support for the Object Transfer Service Client.
+
+if BT_OTC
+
+config BT_DEBUG_OTC
+	bool "Object Transfer Client debug"
+	depends on BT_AUDIO_DEBUG
```

Comment:
Should not depend on BT_AUDIO_DEBUG

Reply:
I see your point, I think.  As a general service/client, it should not depend upon anything AUDIO.
Added a TODO to remove this when OTC is moved out of the audio directory